frontend/src/app/chart/chart.component.ts

```diff
@@ -161,70 +184,119 @@ export class ChartComponent implements OnInit, OnDestroy {
       .attr('transform', `translate(${this.chartMargin},0)`)
       .call(d3.axisLeft(this.yScale));
 
-    this.svgLine = this.svgChart.append('g');
+    this.svgChart
+      .append('g')
+      .append('text')
+      .classed('x-axis-legend', true)
+      .attr('text-anchor', 'left')
+      .attr('alignment-baseline', 'middle')
+      .attr('font-size', '10px')
+      .attr('opacity', 0.5)
+      .attr('x', this.chartWidth / 2)
+      .attr('y', this.chartHeight - this.chartMargin / 2)
+      .text('Time (m:s.ms)');
 
-    // Brush functionality
-    this.brush = d3
-      .brushX()
-      .extent([
-        [this.chartMargin, this.chartMargin],
-        [
-          this.chartWidth - this.chartMargin,
-          this.chartHeight - this.chartMargin,
-        ],
-      ])
-      .on('end', this.interactChart.bind(this));
+    this.svgChart
+      .append('g')
+      .append('text')
+      .classed('y-axis-legend', true)
+      .attr('text-anchor', 'left')
+      .attr('transform', 'rotate(-90)')
+      .attr('alignment-baseline', 'middle')
+      .attr('font-size', '10px')
+      .attr('opacity', 0.5)
+      .attr('x', -this.chartMargin * 2)
+      .attr('y', this.chartMargin + this.chartPadding * 2)
+      .text('Power (mW)');
+
+    this.svgLine = this.svgChart.append('g');
 
     const setFocus = (container: d3.ContainerElement) => {
```

Comment:
This method is defined inside of loadData method right? Any reasons for doing this specifically? It seems to me that setFocus has no relationship with loadData right?

Reply:
setFocus is defined in initChart, it's to be bind with the brush event and would never be used elsewhere, so I thought I could just defined it here...